root/types.js

```diff
@@ -268,6 +270,19 @@ declare type OptionTreeT =
       +description: string,
     |};
 
+/*
+ * See http://search.cpan.org/~lbrocard/Data-Page-2.02/lib/Data/Page.pm
+ * Serialized in MusicBrainz::Server::TO_JSON.
+ */
+declare type PagerT = {|
+  +current_page: number,
+  +first_page: 1,
+  +last_page: number,
+  +next_page: number | null,
```

Comment:
Can we decide on some consistent repr. for nullable fields? either `?type` or `type | null`, I personally prefer the former

Reply:
Well, Perl's JSON module will always serialize `undef` as `null` (JSON doesn't have `undefined` like JavaScript does). And I haven't come across a case yet where I wanted to allow a property to be either null or undefined, so I tend to avoid `?type`.